### PR TITLE
Update DG with implementation of customer add

### LINKED ISSUE #176: Update DG - Customer Add

### PR DESCRIPTION
**Pull request by zhonghan721**
Closes #176 

image

image

### CODE REVIEW

File: docs/diagrams/CustomerAddActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+skinparam ArrowFontSize 12
+!pragma useVerticalIf on
+start
+:User executes customer add command;
+:Parse command string;
+
+if () then ([else])
+    :ParseException: Invalid Command Format;
+    stop
+([all fields are filled and valid]) elseif () then ([else])
```
Comment (jianyangg) [inline docs/diagrams/CustomerAddActivityDiagram.puml]: Could consider adding a space between `(` and `[` such that the square
bracket does not overlap with the lines in the activity diagram.

File: docs/diagrams/CustomerAddActivityDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    :CommandException: User Not Authenticated;
+    stop
+([user is logged in]) elseif () then ([else])
```
Comment (jianyangg) [inline docs/diagrams/CustomerAddActivityDiagram.puml]: Same as above.

**Comment by zhonghan721** [comment]
telegram-cloud-photo-size-5-6186167707578252203-y

**Review by juliusgambe — COMMENTED** [review]
For the sequence diagram, is hasPerson supposed to return anything?

**Review by B-enguin — APPROVED** [review]
LGTM

**Pull request merged**